server/WebAPI/Services/AuthService.cs

```diff
@@ -8,33 +8,33 @@
 
 namespace WebAPI.Services
 {
-    public class AuthService
+  public class AuthService
+  {
+    private readonly IConfiguration _configuration;
+
+    public AuthService(IConfiguration configuration)
     {
-        private readonly IConfiguration _configuration;
-        
-        public AuthService(IConfiguration configuration)
-        {
-            _configuration = configuration;
-        }
+      _configuration = configuration;
+    }
 
-        public string MakeToken(User user)
-        {
-            var tokenHandler = new JwtSecurityTokenHandler();
-            var key = Encoding.ASCII.GetBytes(_configuration["Jwt:Key"]);
-            var tokenDescriptor = new SecurityTokenDescriptor
-            {
-                Subject = new ClaimsIdentity(new[] 
-                {
+    public string MakeToken(User user, bool oneDay = true)
```

Comment:
nobody will remember that second parameter is "oneDay". may be IDE will hive hint. But even with hint you'll guess that "true" is for "oneDay" and what is "false" for? for twoDays? oneMonths? oneMinute?
It is much more convenient to have parameter name stating explicitly its purpose. you can have here "timeFrame" or "timePeriod" or "expirationInterval" with enum of plain seconds after which token will expire
MakeToken(User user, int lifetime)

Reply:
I got it, I'll fix it